src/lib.rs

```diff
@@ -357,10 +357,13 @@ pub enum MouseCursor {
 
 /// Describes if the Window is in one of the fullscreen modes
 #[derive(Clone)]
-pub enum FullScreenState {
+pub enum FullScreen {
+    /// Non-fullscreen window with normal decorations
     None,
-    Windowed,
-    Exclusive(MonitorId),
+    /// Set fullscreen in whatever monitor the window happens to be right now
```

Comment:
I think that's a bad design. The user should just have a way to retrieve the monitor of the window instead.

Reply:
In that case we can just get rid of the enum and do `.set_/.with_fullscreen(Option<MonitorId>)` and do a `Window::get_monitor()`. How does that sound?

Reply:
I guess it's good.